bootstrap.py

```
@@ -0,0 +1,256 @@
+- 이 파일은 오케스트레이션(의존성 설치, 서버 실행/종료)만 담당한다.
+"""
+
+from __future__ import annotations
+
+import os
+import shutil
+import subprocess
+import sys
+import time
+from pathlib import Path
+from typing import IO, Iterable, List, Tuple
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)